src/measure_theory/integral/integral_eq_improper.lean

```diff
@@ -482,6 +576,29 @@ lemma integrable_on_Ioi_of_interval_integral_norm_tendsto (I a : ℝ)
 let ⟨I', hI'⟩ := h.is_bounded_under_le in
   integrable_on_Ioi_of_interval_integral_norm_bounded I' a hfi hb hI'
 
+lemma integrable_on_Ioc_of_interval_integral_norm_bounded {I a₀ b₀ : ℝ}
+  (hfi : ∀ i, integrable_on f $ Ioc (a i) (b i))
+  (ha : tendsto a l $ 𝓝 a₀) (hb : tendsto b l $ 𝓝 b₀)
+  (h : ∀ᶠ i in l, (∫ x in Ioc (a i) (b i), ∥f x∥ ) ≤ I) : integrable_on f (Ioc a₀ b₀) :=
```

Comment:
The usage of `I` for the upper bound is consistent with all the existing code elsewhere in that file (e.g. [here](https://leanprover-community.github.io/mathlib_docs/measure_theory/integral/integral_eq_improper.html#measure_theory.ae_cover.integrable_of_integral_bounded_of_nonneg_ae)). It doesn't seem to have been a problem so far, perhaps because the complex numbers play no role here.

Reply:
👍